yt/visualization/plot_window.py

```diff
@@ -2066,6 +2066,8 @@ def plot_2d(ds, fields, center='c', width=None, axes_unit=None,
         axis = "z"
     elif ds.geometry == "cylindrical":
         axis = "theta"
+    elif ds.geometry == "spherical":
+        axis = "phi"
```

Comment:
While we're at it can you add an `else` block that raises a `NotImplementedError` so future users unsupported geometries get a better error message:

```
else:
    raise NotImplementedError("plot_2d does not yet support datasets with {} geometries".format(ds.geometry)")
```

Reply:
excellent suggestion